core-bundle/src/Command/DebugDcaCommand.php

```diff
@@ -15,6 +15,7 @@
 use Contao\CoreBundle\Framework\ContaoFramework;
 use Contao\DcaLoader;
 use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
```

Comment:
We're throwing `Symfony\Component\Console\Exception\InvalidArgumentException` in some commands and `\InvalidArgumentException` in others. Intentional?

Reply:
No. We should always use the Symfony exception if the invalid argument is a console argument. Can you create a separate PR for this please?

Reply:
Will do.

Reply:
see https://github.com/contao/contao/pull/2581